--- kstars-15.12.2/kstars/tools/horizonmanager.cpp~	2016-01-29 12:17:25.000000000 +0000
+++ kstars-15.12.2/kstars/tools/horizonmanager.cpp	2016-03-03 09:46:43.373213396 +0000
@@ -192,7 +192,7 @@
         az  = dms::fromString(regionItem->child(i, 1)->data(Qt::DisplayRole).toString(), true);
         alt = dms::fromString(regionItem->child(i, 2)->data(Qt::DisplayRole).toString(), true);
 
-        if (isnan(az.Degrees()) || isnan(alt.Degrees()))
+        if (std::isnan(az.Degrees()) || std::isnan(alt.Degrees()))
             return false;
 
         p.setAz(az);
@@ -534,9 +534,9 @@
         dms azAngle  = dms::fromString(parent->child(item->row(),1)->data(Qt::DisplayRole).toString(), true);
         dms altAngle = dms::fromString(parent->child(item->row(),2)->data(Qt::DisplayRole).toString(), true);
 
-        if (isnan(azAngle.Degrees()))
+        if (std::isnan(azAngle.Degrees()))
             azOK = false;
-        if (isnan(altAngle.Degrees()))
+        if (std::isnan(altAngle.Degrees()))
             altOK = false;
 
         if ( (item->column() == 1 && azOK == false) || (item->column() == 2 && altOK == false) )
